stock_picking_partner_language/picking_report.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <report 
```

Comment:
@pedrobaeza in our expirience, customer might need 2 reports: one with the user language for "internal use" and one with the partner language.
What do you think about it?